docs/userguide/install.md

```diff
@@ -135,12 +135,13 @@ Example invocation:
 cd source
 sqlplus admin/admins_password@database @create_synonyms_and_grants_for_public.sql ut3  
 ```
-To grant utPLSQL to an individual user, execute script `source/create_synonyms_and_grants_for_user.sql`, provide `schema_name` where utPLSQL is installed and `user_name` to grant access for.
+To grant utPLSQL to an individual user, execute scripts `source/create_user_grants.sql` and `source/create_user_synonyms.sql`, provide `schema_name` where utPLSQL is installed and `user_name` to grant access for.
 
 Example invocation:
 ```bash
 cd source
-sqlplus admin/admins_password@database @create_synonyms_and_grants_for_user.sql ut3 hr  
+sqlplus admin/admins_password@database @create_user_grants.sql ut3 hr
```

Comment:
Thats true. It depends on what you will imagine behind "admin user". I will change the description in docs. It can be sysdba as well as ut3 user. It really depends on the who is executing the script.